cmd/execution_builder.go

```diff
@@ -933,6 +948,89 @@ func (exeNode *ExecutionNode) LoadExecutionDataPruner(
 	return exeNode.executionDataPruner, err
 }
 
+func (exeNode *ExecutionNode) LoadObserverCollectionIndexer(
+	node *NodeConfig,
+) (
+	module.ReadyDoneAware,
+	error,
+) {
+	if !node.ObserverMode {
+		return &module.NoopReadyDoneAware{}, nil
+	}
+
+	execDataDistributor := edrequester.NewExecutionDataDistributor()
+
+	executionDataDownloader := execution_data.NewDownloader(exeNode.blobService)
+
+	var heroCacheCollector module.HeroCacheMetrics = metrics.NewNoopCollector()
+	execDataCacheBackend := herocache.NewBlockExecutionData(10, node.Logger, heroCacheCollector)
+
+	// Execution Data cache that uses a blobstore as the backend (instead of a downloader)
+	// This ensures that it simply returns a not found error if the blob doesn't exist
+	// instead of attempting to download it from the network.
```

Comment:
this comment isn't right. it's actually using a downloader (correctly).

Reply:
OK, it was copied from here tho

https://github.com/onflow/flow-go/blob/leo/localnet-test-en/cmd/access/node_builder/access_node_builder.go#L559-L561